### PR TITLE
[15.0][ADD] stock_picking_user_assign: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
Check a picking has not responsible assigned  and  assign one and open it from tree view, kanban view and form view.

cc @Tecnativa

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: stock_picking_user_assign/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def action_assign_to_me(self):
+        if self.filtered("user_id"):
+            raise UserError(_("Picking already responsible assigned"))
```
Comment (rousseldenis) [inline stock_picking_user_assign/models/stock_picking.py]:
```suggestion
            raise UserError(_("Picking has already a responsible assigned"))
```

**Review by rousseldenis — COMMENTED** [review]
Code review.

@sergio-teruel FYI, there is also a module in this related field:

https://github.com/OCA/stock-logistics-workflow/tree/16.0/stock_picking_start

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by carlosdauden** [comment]
Functionality merged in https://github.com/OCA/stock-logistics-workflow/pull/1554
Can be closed.

**Pull request closed without merging**